docs/GraFx-Studio/guides/template-variables/define/index.md

```diff
@@ -65,34 +69,34 @@ A placeholder for a reference to an image in the (GraFx) Media pool, or other so
 
 ## Default state
 
-The default value your variable with have.
+The default value your template variable with have.
 
 ## Basic properties:
 
 - Name: Cannot be empty, must be unique
-- Type: See [types](#variable-types) above
+- Type: See [types](#types) above
 - [Visible](#visibility-conditions): Enabled by default
-- [Required](#required-variables): Disabled by default
+- [Required](#required-template-variables): Disabled by default
 - Read-only: Disabled by default
 
-### Required variables
+### Required template variables
 
 When checked, the Studio Editor will ask the user to provide a value.
 
 !!! Info
-	When a variable is required and it doesn't have a default value it doesn't show an error upon opening the document.
+	When a template variable is required and it doesn't have a default value it doesn't show an error upon opening the document.
 
 An error is triggered upon these conditions:
 
 - When the input is in focus and left empty.
 - When the user enters a value for the first time, deletes it again (so the input is empty again) the error message will be shown below the input until the error is resolved.
 - When trying to save the document or generate output. A dialog is shown to inform the user that the document can't be saved or exported.
 
-To indicate that a variable is required an asterisk ( * ) is shown next to the variable label.
+To indicate that a template variable is required an asterisk ( * ) is shown next to the label.
 
 ### Visibility conditions
 
-When the "Visible" setting is enabled (is enabled by default), there is an option to add one or more visibility conditions. Only if ALL conditions are met the variable is visible for the end-user.
+When the "Visible" setting is enabled (is enabled by default), there is an option to add one or more visibility conditions. Only if ALL conditions are met the template variable is visible for the end-user.
```

Comment:
We do not have visibility conditions yet. It's also not on the roadmap, so it will probably not arrive any time soon. It was an idea we had, but it's not planned yet.